### ISSUE TITLE
Experimental autofill on geizhals.de

### ISSUE THREAD

**Issue by Nemo64** [bitwarden/browser#1043]
I just had a very strange behaviour when enabling autofilling of passwords on geizhals.de.
![Bildschirmvideo aufnehmen 2019-10-18 um 21 43 46 cut mov](https://user-images.githubusercontent.com/1749936/67125090-4d238b00-f1f4-11e9-89d5-df759dc0ce99.gif)

The page will just continue to reload until you manually stop it or the webserver blocks you because of unusual traffic (rightfully so).

I'm not really sure how to debug that.

**Comment by kspearrin** [comment]
What kind of data is stored in the item you have associated with this website? I see there is 1 item in your vault for it. Is it just username/password or is there more to it?

**Comment by Nemo64** [comment]
<img width="367" alt="Bildschirmfoto 2019-10-18 um 22 23 56" src="https://user-images.githubusercontent.com/1749936/67125896-18b0ce80-f1f6-11e9-846a-5a25f0f4c9b3.png">
You are totally right. It seems I managed to store a lot of filters while saving the password. And changing these filters will trigger page reloads...

I can't really tell you if I created the password using BitWarden or imported it from 1Password.
Maybe just limit the additional field storage to the actually submitted form?
Anyways, thanks!
